### ISSUE TITLE
TLP over Tuned for Laptop

### ISSUE THREAD

**Issue by JohnyAKAMystery** [ublue-os/bazzite#1011]
### Describe the package

I would like to talk about TLP over TuneD. If for nothing else, at least to get a better understanding of the reason behind said choice.

I can see how TuneD might be a more powerful choice if making power profiles for known hardware such as Handheld Gaming devices, where a profile can be tweaked expertly for maximum performance/power saving, but for a general purpose use, I would argue it is inferior to TLP.

For a project that aims to be as OoTB friendly as possible, even if TuneD when properly used can probably be quite powerful, that is a more advanced thing. 

I, as well as a few friends of mine, have seen better results by simply disabling TuneD and layering TLP, ticking some things in TLP-UI, than with messing around with the TuneD Manager to attempt to get a decent profile (and yes, the results from TLP are far superior to the prebuilt Power-Saving profile). Somewhere from 50%-100% more power when idling (according to PowerTop's estimate), and it's also noticable during normal use that it depletes less quickly.

tl;dr: for improving Bazzite's OoTB experience, which is the whole point, I'd argue TLP is a better choice than TuneD. I can still understand TuneD for the Handheld images.

Thank you for your attention and all the work put into the Bazzite experience, it's been really good besides this one thing!

-Johny

### Image

Desktop
